firmware/lib/simplecan/SimpleCan.cpp

```diff
@@ -0,0 +1,205 @@
+#include "SimpleCan.h"
```

Comment:
Add MIT license and reference to original: https://github.com/owennewo/youtube_can/tree/main/canfd_stm32